packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -151,7 +151,7 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
         isOrg: input.isOrg,
       });
     });
-    sendEmails(sendVerifEmailsPromises);
+    await sendEmails(sendVerifEmailsPromises);
```

Comment:
I think this was causing a race condition that caused to report a wrong membership count